skimage/transform/_hough_transform.pyx

```diff
@@ -314,7 +314,8 @@ def _hough_line(cnp.ndarray img,
 
 def _probabilistic_hough_line(cnp.ndarray img, int threshold,
                               int line_length, int line_gap,
-                              cnp.ndarray[ndim=1, dtype=cnp.double_t] theta):
+                              cnp.ndarray[ndim=1, dtype=cnp.double_t] theta,
+                              seed=None):
```

Comment:
I don't know Cython realy well, but pretty sure there should be a more "cythonic" way of doing  this (something like `int seed...`). Otherwise, IIRC, `seed` is an `object`.

P.S> Everything else in this PR LGTM.